uwg_2023/uwg_2023_stac.ipynb

```diff
@@ -0,0 +1,850 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "edb65a37-041e-4bb0-9850-167ff9abcf2e",
+   "metadata": {},
+   "source": [
+    "# STAC Catalog for the Global Coherence Dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fce2004e-9b41-40ce-a8fc-0e5f14c92363",
+   "metadata": {},
+   "source": [
+    "## 1. The Why"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4cf31097-81d8-44f7-8272-011116eeffe1",
+   "metadata": {},
+   "source": [
+    "SpatioTemporal Asset Catalogs are an emerging standard for geospatial metadata. They are the de-facto standard for commercial SAR companies, and even CMR is working on a STAC interface. Since many of users are becoming accustomed to using STAC catalogs, ASF's Tools team has been building experience working with this technology. To that end, we have created a STAC catalog and STAC API for Dr. Joseph Kellendorfer's [Global Seasonal Sentinel-1 Interferometric Coherence and Backscatter Data Set](https://registry.opendata.aws/ebd-sentinel-1-global-coherence-backscatter/). This allows users to access this dataset in a whole new way!"
```

Comment:
"and even CMR is working on a STAC interface" feels a touch too dismissive
```suggestion
    "SpatioTemporal Asset Catalogs are an emerging standard for geospatial metadata. They are the de-facto standard for commercial SAR companies, and even CMR maintains a STAC interface. Since many of users are becoming accustomed to using STAC catalogs, ASF's Tools team has been building experience working with this technology. To that end, we have created a STAC catalog and STAC API for Dr. Joseph Kellendorfer's [Global Seasonal Sentinel-1 Interferometric Coherence and Backscatter Data Set](https://registry.opendata.aws/ebd-sentinel-1-global-coherence-backscatter/). This allows users to access this dataset in a whole new way!"
```